web/app/components/comment/comment.jsx

```diff
@@ -560,3 +603,26 @@ function formatTime(time) {
 
   return `${date} at ${hours}:${mins}`;
 }
+
+function getBlockArray() {
```

Comment:
No need in a function here. It could be declared just as constant in a corresponding file.